### PR TITLE
fix plugins/http-log

### PR DESCRIPTION
**Pull request by WALL-E**
### Summary
support set parsed_url.query and content-type

* fix
  * content-type
* add
  * parsed_url.query

### CODE REVIEW

File: kong/plugins/http-log/handler.lua [hunk 1]
```diff
@@ -14,10 +14,16 @@ local HTTPS = "https"
 -- @param `parsed_url` contains the host details
 -- @param `message`  Message to be logged
 -- @return `body` http payload
```
Comment (Tieske) [inline kong/plugins/http-log/handler.lua]: I think these comments need to be updated, they do not match the code anymore (not caused by this PR, but would be nice to fix that in one go). And some others in the code too.

File: kong/plugins/http-log/schema.lua [hunk 2]
```diff
@@ -2,6 +2,7 @@ return {
   fields = {
     http_endpoint = { required = true, type = "url" },
     method = { default = "POST", enum = { "POST", "PUT", "PATCH" } },
+    content_type = { default = "application/json", enum = { "application/json", "application/x-www-form-urlencoded" } },
```
Comment (Tieske) [inline kong/plugins/http-log/schema.lua]: this should not be part of the http-logger, as it only does json. So here it should stay, but as an enum with only one possible value; `application/json`.

**Comment by WALL-E** [comment]
@Tieske  PR updated

**Comment by WALL-E** [comment]
Already moved to  #1696

File: kong/plugins/http-log/schema.lua (hunk 2 above)
Comment (dnirchi) [reply kong/plugins/http-log/schema.lua]: Is it possible to override the 'application/json' default without forking and creating a custom HTTP-LOG plugin?

**Pull request closed without merging**